Packs/Wiz/Integrations/Wiz/Wiz.py

```diff
@@ -1445,115 +2482,106 @@ def main():
 
         elif command == 'wiz-reject-issue':
             demisto_args = demisto.args()
-            issue_id = demisto_args.get(WizInputParam.ISSUE_ID)
+            resource_id = demisto_args.get(WizInputParam.ISSUE_ID)
             resolution_reason = demisto_args.get(WizInputParam.REJECT_REASON)
             resolution_note = demisto_args.get(WizInputParam.REJECT_NOTE)
-            issue_response = reject_issue(
-                issue_id=issue_id,
+            copy_mutation_response = reject_issue(
+                issue_id=resource_id,
                 reject_reason=resolution_reason,
                 reject_comment=resolution_note
             )
-            command_result = CommandResults(readable_output=issue_response, raw_response=issue_response)
+            command_result = CommandResults(readable_output=copy_mutation_response, raw_response=copy_mutation_response)
             return_results(command_result)
 
         elif command == 'wiz-reopen-issue':
             demisto_args = demisto.args()
-            issue_id = demisto_args.get(WizInputParam.ISSUE_ID)
+            resource_id = demisto_args.get(WizInputParam.ISSUE_ID)
             reopen_note = demisto_args.get(WizInputParam.REOPEN_NOTE)
-            issue_response = reopen_issue(
-                issue_id=issue_id,
+            copy_mutation_response = reopen_issue(
+                issue_id=resource_id,
                 reopen_note=reopen_note
             )
-            command_result = CommandResults(readable_output=issue_response, raw_response=issue_response)
+            command_result = CommandResults(readable_output=copy_mutation_response, raw_response=copy_mutation_response)
             return_results(command_result)
 
         elif command == 'wiz-resolve-issue':
             demisto_args = demisto.args()
-            issue_id = demisto_args.get(WizInputParam.ISSUE_ID)
+            resource_id = demisto_args.get(WizInputParam.ISSUE_ID)
```

Comment:
why you don't name this param issue_id? (in all the other functions as well)

Reply:
refactor mistake, reverting